pkg/cloudprovider/aws/aws.go

```diff
@@ -175,8 +478,8 @@ func (aws *AWSCloud) Clusters() (cloudprovider.Clusters, bool) {
 }
 
 // TCPLoadBalancer returns an implementation of TCPLoadBalancer for Amazon Web Services.
-func (aws *AWSCloud) TCPLoadBalancer() (cloudprovider.TCPLoadBalancer, bool) {
-	return nil, false
+func (self *AWSCloud) TCPLoadBalancer() (cloudprovider.TCPLoadBalancer, bool) {
```

Comment:
I agree with Justin, but I am obligated to post:

https://github.com/golang/go/wiki/CodeReviewComments#receiver-names